source/elements/oneMKL/source/domains/rng/onemkl-rng-overview.rst

```diff
@@ -35,42 +35,10 @@ In computational statistics, random variate generation is usually made in two st
       in order to generate (or imitate) random variates and random
       vectors from arbitrary distributions.
 
```

Comment:
Maybe to add short description about manual offload and device API here

Reply:
added, thanks for help